src/migration/1724168427191-UserLanguage.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class UserLanguage1724168427191 implements MigrationInterface {
+  name = 'UserLanguage1724168427191';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "user" ADD "language" text`,
```

Comment:
We don't want to set a default?

Reply:
I was thinking we might want to differentiate users that selected language and those legacy ones that did not yet? So we can (for example) run a campaign to update language at later date?

Reply:
Either way in API users will fallback to english when nothing is set

Reply:
Ah yeah that's good :)